### DISCUSSION TITLE
Exclude All but One File in Backup

### DISCUSSION THREAD

**Discussion by ClemKristian** [pterodactyl/panel#3407] [Support] [answered]
Is there a way to exclude everything else in a folder except one file? I want to exclude all log files except the latest log. Is there any wildcard I can apply that would help me accomplish that? I have tried using "!", getting the idea from https://superuser.com/questions/430918/how-can-delete-all-files-except-one-from-a-directory.

**Comment by trenutoo** [comment, marked as answer]
The format used is the same as `.gitignore`. So the example below would first ignore all files with a wildcard `*` and then unignore a specific file or a folder with `!`.

```
*
!testfile
```
You can whitelist multiple files, folders, or subtrees by adding more with `!`

**Reply by ClemKristian** [reply]
That seemed to do the trick. Thanks for the reply. In my case, for Minecraft I used, 
`logs/*`
and then 
`!latest.log`
